src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -46,6 +46,11 @@
 
     LocalDateTime createdTime;
 
+    @Lob
```

Comment:
can you remove this? This caused some problems when used in Post for displaying images